cirq-core/cirq/experiments/two_qubit_xeb.py

```diff
@@ -156,9 +158,163 @@ def plot_histogram(self, ax: Optional[plt.Axes] = None, **plot_kwargs) -> plt.Ax
+            }
+        )
+
+    def _target_errors(
+        self, target_error: str
+    ) -> Mapping[Tuple['cirq.GridQubit', 'cirq.GridQubit'], float]:
+        error_funcs = {
+            'pauli': self.inferred_pauli_error,
+            'decay_constant': self.inferred_decay_constant,
+            'xeb': self.inferred_xeb_error,
+        }
+        return error_funcs[target_error]()
+
+    def plot_heatmap(
+        self, target_error: str = 'pauli', ax: Optional[plt.Axes] = None, **plot_kwargs
+    ) -> plt.Axes:
+        """plot the heatmap of the target errors.
+
+        Args:
+            target_error: The error to draw. Must be one of 'xeb', 'pauli', or 'decay_constant'
+            ax: the plt.Axes to plot on. If not given, a new figure is created,
+                plotted on, and shown.
+            **plot_kwargs: Arguments to be passed to 'plt.Axes.plot'.
+        """
+        show_plot = not ax
+        if not isinstance(ax, plt.Axes):
+            fig, ax = plt.subplots(1, 1, figsize=(8, 8))
+        heatmap_data = cast(
+            Mapping[Tuple['cirq.GridQubit', ...], float], self._target_errors(target_error)
+        )
+
+        name = f'{target_error} error' if target_error != 'decay_constant' else 'decay constant'
+        ax.set_title(f'device {name} heatmap')
+
+        vis.TwoQubitInteractionHeatmap(heatmap_data).plot(ax=ax, **plot_kwargs)
+        if show_plot:
+            fig.show()
+        return ax
+
+    def plot_histogram(
+        self,
+        target_error: str = 'pauli',
+        ax: Optional[plt.Axes] = None,
+        kind: str = 'two_qubit',
+        **plot_kwargs,
+    ) -> plt.Axes:
+        """plot a histogram of target error.
+
+        Args:
+            target_error: The error to draw. Must be one of 'xeb', 'pauli', or 'decay_constant'
+            kind: Whether to plot the single-qubit RB errors ('single_qubit') or the
+                two-qubit inferred errors ('two_qubit') or both ('both').
+            ax: the plt.Axes to plot on. If not given, a new figure is created,
+                plotted on, and shown.
+            **plot_kwargs: Arguments to be passed to 'plt.Axes.plot'.
+
+        Raises:
+            ValueError: If
+                - `kind` is not one of 'single_qubit', 'two_qubit', or 'both'.
+                - `target_error` is not one of 'pauli', 'xeb', or 'decay_constant'
+                - single qubit error is requested and `target_error` is not 'pauli'.
+        """
+        if kind not in ('single_qubit', 'two_qubit', 'both'):
+            raise ValueError(
+                f"kind must be one of 'single_qubit', 'two_qubit', or 'both', not {kind}"
+            )
+        if kind != 'two_qubit' and target_error != 'pauli':
+            raise ValueError(f'{target_error} is not supported for single qubits')
+        fig = None
+        if ax is None:
+            fig, ax = plt.subplots(1, 1, figsize=(8, 8))
+
+        alpha = 1 / (1 + int(kind == 'both'))
```

Comment:
Nit - let us save some brain cycles -
```suggestion
        alpha = 0.5 if kind == 'both' else 1.0
```